spec/support/shared_examples_for_otp_forms.rb

```diff
@@ -1,8 +1,8 @@
 shared_examples_for 'an otp form' do
-  it 'disables autocomplete' do
+  it 'marks field to number type' do
     render
 
-    expect(rendered).to have_css('input[autocomplete=off]')
+     expect(rendered).to have_css('input[type=number]')
```

Comment:
Looks like this is causing issues with the personal key entry form test:

```
Failures:

  1) two_factor_authentication/personal_key_verification/show.html.slim behaves like an otp form marks field to number type
     Failure/Error: expect(rendered).to have_css('input[type=number]')
       expected to find css "input[type=number]" but there were no matches
     Shared Example Group: "an otp form" called from ./spec/views/two_factor_authentication/personal_key_verification/show.html.slim_spec.rb:12
     # ./spec/support/shared_examples_for_otp_forms.rb:5:in `block (2 levels) in <top (required)>'
```

Tbh I am not sure I see the value in this test. I'd be fine pulling the whole thing.

Additionally it looks like CodeClimate is complaining about too much indentation here. That is, of course, also resolved by removing this test case.